src/styles/components/_news.scss

```diff
@@ -1,10 +1,11 @@
-.controller-news.action-show {
-  #content div.news {
-    position: relative;
-    margin-bottom: 10px;
-    border-radius: 3px;
-    background-color: $color-history-bg;
-    border: 1px solid $color-history-border;
-    padding: $histry-padding;
+.controller-news.action-show div.news {
+  margin: 20px 0;
```

Comment:
> 2. Changed save button to include margin

I think you added the style `margin: 20px 0;` for this change, but as a result, the spacing between the news title and body has also become larger. This spacing is not consistent with the forum or issues.

(Left: News detail, Right: Forum message detail) <img width="1388" height="286" alt="Image" src="https://github.com/user-attachments/assets/ed78e1ea-2517-4a05-8a39-9e7e754b1c0a" />

How about applying `margin-bottom: 20px` to `#edit-news` and `#add-news` instead?

Reply:
@hidakatsuya 
Thank you for your feedback. I have changed the selector based on your comments.

div#add-news 
<img width="1508" height="601" alt="screenshot 2025-08-22 15 45 17" src="https://github.com/user-attachments/assets/0fb22267-112b-422a-a7c1-f5c9c155b8e1" />

div#edit-news
<img width="1508" height="601" alt="screenshot 2025-08-22 15 45 26" src="https://github.com/user-attachments/assets/bdc17f6c-4bc0-41c2-9ada-0c97dd810a3d" />